lib/router.js

```diff
@@ -236,6 +236,11 @@ Router.prototype._handleSwaggerPathSpec = function(node, pathspec,
                 specRoot.paths[prefixPath][methodName] = method;
             }
 
+            if (node.value.methods[methodName]) {
+                throw new Error('Trying to re-define existing method '
+                    + node.value.path + ':' + methodName);
+            }
+
```

Comment:
Before the fix we were failing way earlier, before loading modules and sub specs, but it does not matter how early or late to fail the startup.